.changeset/young-carpets-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Added opt-in ability to evict entities from the catalog whose provider is no longer configured
```

Comment:
Best link to docs or mention the new config key here

Reply:
Resolved in [ca9c51b](https://github.com/backstage/backstage/pull/28971/commits/ca9c51ba1bb30ea326e323c37e1ba469222a2452)